src/libraries/Microsoft.PowerFx.LanguageServerProtocol/Protocol/WorkspaceEdit.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Microsoft.PowerFx.LanguageServerProtocol.Protocol
+{
+    /// <summary>
+    /// Workpsace edit object model.
+    /// </summary>
+    public class WorkspaceEdit
+    {
+        /// <summary>
+        /// ctor
+        /// </summary>
+        public WorkspaceEdit()
+        {
+            Changes = new Dictionary<string, TextEdit[]>();
+        }
+
+        /// <summary>
+        /// Gets or sets suggested changes.
+        /// </summary>
+        public Dictionary<string, TextEdit[]> Changes { get; set; }
```

Comment:
>string [](http://example.com/codeflow?start=26&length=6)

what is the key?

Reply:
key is document uri https://microsoft.github.io/language-server-protocol/specifications/specification-3-17/#workspaceEdit

Reply:
add that as a comment in the code?